nextjs-nabil/styles/modal.module.scss

```diff
@@ -0,0 +1,52 @@
+$primarycolor: #a94949;
```

Comment:
this could've been a lot better imported from a file like Colors.scss where it contains a centralized location for all the color themes. This will avoid having inconsistency in colors and manually changing each file if a main theme color was changed.

Reply:
Noted